assets/css/styles.css

```diff
@@ -2561,3 +2563,100 @@ code.highlighter-rouge{
 }
 
 
+/* Search hits stylling */
+.ais-hits--item{
+  padding: 2% 5% 0;
+
+}
+
+.stats{
+  padding: 5% 5% 0;
+}
+
+#category-widget{
+  padding-top:2%;
+
+}
+
+.ais-RefinementList-checkbox {
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    appearance: none;
+    background-color: rgba(65,66,71,.08);
+    border: none;
+    border-radius: 2px;
+    height: 1rem;
+    margin: 0 1rem 0 0;
+    position: relative;
+    width: 1rem;
+}
+
+label{
+  font-weight: normal !important;
+}
+
+.dropdown-menu{
+  right:auto !important;
+  left:auto;
+  min-width: 600px !important;
+  background-color: #FFF !important;
+
+}
+.scrollable-menu {
+    height: auto;
+    max-height: 500px;
+    overflow-x: hidden;
+}
+.dropdown-menu:before{
+  position: fixed !important;
+}
```

Comment:
I am not conformtable with overidding existing classes.

Reply:
Can i create a new class for the styles? For the dropdown

Reply:
styled the dropdown using an id

Reply:
you choice of variable name is ambiguous. Call it something meaningful besides #theDropDown. Also check the classes you are overriding. Do a search in the file for that class and edit there.